server/reports/invoice_landscape/2_3_0/manifest.json

```diff
@@ -0,0 +1,17 @@
+{
+    "is_custom": false,
+    "version": "2.3.0",
+    "code": "invoice-landscape",
+    "context": "REPORT",
```

Comment:
I think this invoice one isn't part of the REPORTS section, so we'll put this in at a later date...
(This would show up under invoice I think)